src/main/com/aca/homework/week9/calculator/StringSupplier.java

```diff
@@ -0,0 +1,5 @@
+package com.aca.homework.week9.calculator;
+
+public interface StringSupplier {
```

Comment:
please create another implementation that ask an input from a user

Reply:
Can't I just use anonymous class with the ```get``` method implemented like this: ```return new Scanner(System.in).nextLine()```?